### DISCUSSION TITLE
Feature feedback.

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#87] [Polls]
Labs can benefit from your feedback.  If you don't see you favorite listed, please add in the comments.

**Comment by 50ShadesOfDamon** [comment]
couple requests:
 
can you please resolve the issue where the camera can’t connect to Wi-Fi with a landing page. Whether it be by utilizing the phones already active network connection or through other means. It makes no sense for those of us who travel to hotels a lot etc.. that we don’t have the ability to upload. A hotspot isn’t a fix as it would eat up 50gb in one upload.

I do driving videos and I know one of the natural things in the camera is for it to drift left or right (depending on long turns) or up and down when cars stopping and going. This exposes the hood on the car. Tried various ways to make this stop while keeping some type of stability and can’t fix that. Sort of like gimbal drift. Other cameras are able to resolve this somehow, wondered if you could also.

while the QR code is cool for the betas, having ability to change in camera would be nice also.
Is there a way for light sensor capture like for lightning etc? For video and photo? iPhone has this ability w software wondered about for the camera? :)

oh, and more refined stickers as some call them? Having to pay 300.00 for cleaner telemetry data software is nuts. The ones you have are okay but look a bit Janky. Some more professional looking ones would be great!
 
thanks!

Translate with Google

Choose a language

**Reply by dnewman-gpsw** [reply]
I added a feature requests section.  This is not a feedback to the above poll.

Unfortunately most aren't addressable through Labs. Labs is for firmware tweaks, mainly for camera control.  Most of these should be directed here https://gopro.com/en/us/info/static/feedback as that will go more to main team.

WiFi connectivity in hotels is not within Labs team control. 

Hypersmooth settings controls for driving would be nice. Not sure Labs can address this, the magic of HS is hard to hack. 

QR Codes enable Labs, moving Labs features to the UI is what will happen if the feature goes mainstream.  Labs only works as we don't have to modify the UI.

Sticker are in post, definitely not Labs. Send the camera team feedback for the stickers you want.

**Comment by fxstein** [comment]
Missing the single most important feature:

Setting time & date and doing a timecode sync for the cameras. This is the feature we have to use daily and multiple times as cameras crash and lose their time lock.

**Comment by xixar** [comment]
I hope the GoPro 11 screen can display the large crosshair grid. 😅
It helps to center the scene.
Also, I think it will be helpful when shooting architecture and landscapes.
Thanks.

![crosshair](https://user-images.githubusercontent.com/48396675/201486914-094af3ad-7a6f-4a2c-a8f9-3b62f1e75b61.jpg)

**Reply by dnewman-gpsw** [reply]
The HERO11 has the new aspect ratio guide feature, which can be used to generate a crosshair.  This was in the 1.20.70 version released a few days ago.
![image](https://user-images.githubusercontent.com/10856084/201487712-054deeb7-1b62-4893-841a-60cc22a228d0.png)
This is the command I used.
![image](https://user-images.githubusercontent.com/10856084/201487752-36003e2c-b676-4655-98a3-b4443a3054e4.png)

**Reply by xixar** [reply]
Oh, I installed 1.20.70 Firmware yesterday, but I didn't know there was a "GUID" command.
Thank you for the information.
By the way, can you please explain how 200.0,0.01 makes crossshare?

**Reply by dnewman-gpsw** [reply]
GUID is explained here https://gopro.github.io/labs/control/extensions/

Normally it lists aspect ratios like 1.777 and 2.35, but to fake the cross hairs, I used the widest and narrowest aspect ratios the control supports: 200 (very wide) and 0.01 (very tall.)

**Reply by xixar** [reply]
Thanks again for the detailed explanation!
